sw/device/lib/base/multibits_asm.h

```diff
@@ -0,0 +1,38 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_BASE_MULTIBITS_ASM_H_
+#define OPENTITAN_SW_DEVICE_LIB_BASE_MULTIBITS_ASM_H_
+
+/**
+ * Multi-bit boolean values for use in assembly code.
+ *
+ * Please use `multibits.h` instead when writing C code.
```

Comment:
Do you think we could merge these and condition onf `_ASSEMBLY_` or whatever the define the assembler sets is?